pkg/apis/numaflow/v1alpha1/monitor_container.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+func createMonitorContainer(req getContainerReq) corev1.Container {
+	return containerBuilder{}.init(req).name(CtrMonitor).asSidecar().command(NumaflowRustBinary).args("--monitor").build()
```

Comment:
Remove volumeMount from this function, because the volumeMounts contains one item is only supposed to be for the numa container.


```
volumeMounts := []corev1.VolumeMount{{Name: varVolumeName, MountPath: PathVarRun}}
```